public/app/plugins/panel/canvas/editor/options.ts

```diff
@@ -129,6 +141,27 @@ export const optionBuilder: OptionSuppliers = {
     });
   },
 
+  addRadius: (builder, context) => {
+    const category = ['Radius'];
+    builder.addCustomEditor({
+      category,
+      id: 'radius',
+      path: 'radius',
+      name: 'Radius',
+      editor: ScaleDimensionEditor,
```

Comment:
Only allow selection of numeric based fields 
![Screenshot 2024-03-22 at 11 20 49 AM](https://github.com/grafana/grafana/assets/22381771/1c1fc7b1-46c1-4cea-9656-60c5ca8f5719)

Reply:
Created a follow-up issue for this here: https://github.com/grafana/grafana/issues/85019